src/chaosbag/BlessCurseManager.ttslua

```diff
@@ -156,16 +142,26 @@ end
 
 -- context menu function 2
 function doReset(color)
-    -- delete previously pulled out tokens by the token arranger
-    tokenArrangerApi.deleteCopiedTokens()
-
     playerColor = color
     numInPlay = { Bless = 0, Curse = 0 }
     tokensTaken = { Bless = {}, Curse = {} }
     initializeState()
     tokenArrangerApi.layout()
 end
 
+-- removing tokens that were 'taken'
+function removeType(type)
```

Comment:
I think it's hard to tell what this is doing by the name.  Maybe cleanUpTaken()?